inspirehep/modules/disambiguation/config.py

```diff
@@ -25,4 +25,16 @@
 from __future__ import absolute_import, division, print_function
 
 
-DISAMBIGUATION_SAMPLED_PAIRS_SIZE = 1000000
+DISAMBIGUATION_SAMPLED_PAIRS_SIZE = 12 * 100000
```

Comment:
I am not sure why this is in a separate config file, and not ext.py.

Reply:
Because it can be: it's actually bad that configuration has to live in `ext.py` just because they need to have access to an application variable in order to know their value.

Reply:
In fact, defaults for future configuration options that you are going to add (like the number of jobs to use during training or prediction) should probably be added here.